std/bitmanip.d

```diff
@@ -121,15 +119,11 @@ private template createAccessors(
         else
         {
             // getter
-            enum createAccessors = "@property "~T.stringof~" "~name~"() @safe pure nothrow @nogc const { auto result = "
-                ~"("~store~" & "
-                ~ myToString(maskAllElse) ~ ") >>"
-                ~ myToString(offset) ~ ";"
-                ~ (T.min < 0
```

Comment:
Removing this line is almost certainly a cause of red pipelines in this PR too.  For instance `enum E { a, b, c }` is typed "int", and `E : 2` does not fit all values unless you treat it as being logically unsigned.

Reply:
So at the very least you need to have:
```
enum rshiftOp = (T.min < 0 ? ">>= " : ">>>= ");
enum createAccessors = "...."
    ~ "result " ~ rshiftOp ~ myToString(SignShift) ~ ";"
```

Then I imagine all unittests will pass.

Reply:
Interesting. I missed the code special cases enums.